force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -252,11 +317,13 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String createAndGetBookDetails() {
         // Create a Book object with title "A Brief History of Time" and author "Stephen Hawking"
+        Book myBook = new Book('A Brief History of Time', 'Stephen Hawking', 256);
```

Comment:
Here you're forced to add the number of pages in your method call since you added it to your constructor above.